.env.j2

```diff
@@ -0,0 +1,13 @@
+FLASK_ENV=production
+FLASK_APP=todo_app/app
+FLASK_DEBUG=false
+
+SECRET_KEY={{ secret_key }}
+TRELLO_API_KEY={{ trello_api_key }}
+TRELLO_API_TOKEN={{ trello_api_token }}
+
+TRELLO_BOARD_ID=65ca250e1b75ace3f0c03d02
+TRELLO_TO_DO_LIST_ID=65ca250fcecec82c70a14eb4
+TRELLO_DOING_LIST_ID=65ca250f2d1343a290522c86
+TRELLO_DONE_LIST_ID=65ca250f47e27b64ea338731
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)